specification/cognitiveservices/data-plane/LUIS/Authoring/stable/v2.0/LUIS-Authoring.json

```diff
@@ -2495,18 +3155,32 @@
       "get": {
         "description": "Gets the endpoint URLs for the prebuilt Cortana applications.",
         "operationId": "Apps_ListCortanaEndpoints",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiKey"
+          }
+        ],
         "responses": {
           "200": {
             "description": "A personal assistant apps JSON object containing the endpoint URLs for Cortana applications and the user's endpoint keys.",
             "schema": {
               "$ref": "#/definitions/PersonalAssistantsResponse"
             }
           },
-          "default": {
-            "description": "Error Response.",
+          "429": {
+            "$ref": "#/responses/429"
+          },
+          "401": {
+            "$ref": "#/responses/401"
+          },
+          "400": {
```

Comment:
How will this return a 400? The user doesn't send any parameters. Same for the next 3 APIs